bin/runnable-list.js

```diff
@@ -1,40 +1,37 @@
 'use strict'
 
-var program = require('commander')
+const program = require('commander')
 
-var runnable = require('../lib/runnable')
-var Table = require('../lib/table')
-var utils = require('../lib/utils')
+const List = require('../lib/list')
+const Table = require('../lib/table')
+const Utils = require('../lib/utils')
 
 program
   .arguments('[repository]')
   .description('Lists all repositories and services. Specify a repository to list its containers.')
   .parse(process.argv)
 
-var options = {
-  _user: runnable.user,
+const options = {
   repository: program.args.shift()
 }
 
 if (options.repository) {
-  runnable.listContainersForRepository(options)
-    .then(function (results) {
+  List.listContainersForRepository(options)
+    .then((results) => {
       Table.log(results)
     })
-    .catch(utils.handleError)
+    .catch(Utils.handleError)
 } else {
-  runnable.listContainerSummary(options)
-    .then(function (results) {
+  List.listContainerSummary(options)
+    .then((results) => {
       function tableFormatter (obj, cell) {
-        for (var key in obj) {
+        for (const key in obj) {
```

Comment:
That is pretty much the definition of `const`, yes...

On Fri, Mar 11, 2016 at 7:33 AM anandkumarpatel <notifications@github.com>
wrote:

> In bin/runnable-list.js
> <https://github.com/Runnable/cli/pull/13#discussion_r55844189>:
>
> >        function tableFormatter (obj, cell) {
> > -        for (var key in obj) {
> > +        for (const key in obj) {
>
>
> https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75#.etxop9ydk
>
> I think we should use const at all times if the variable is not going to
> be *reassigned* and let all other times
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Runnable/cli/pull/13/files#r55844189>.
>